runtime/ts/relation.ts

```diff
@@ -15,14 +15,20 @@ export class Relation extends Entity {
   entities: Entity[];
 
   constructor(...entities) {
-    super();
+    super(undefined);
     this.entities = entities;
   }
 
+  // should be rawData???
   get data() {
     return this.entities.map(entity => entity[Symbols.identifier].toLiteral());
   }
 
+  // TODO should this return something else?
+  get key() {
```

Comment:
This won't be required once you remove it from the super.

Reply:
done